src/registration.ts

```diff
@@ -39,28 +39,31 @@ class Registration extends Authentication {
 		);
 	};
 
-	submitForm() {
+	submitForm(): IUser {
 		return {
 			username: this._username,
 			password: this._password,
-			email: this._email
+      email: this._email,
+      deleted: false
 		};
 	};
 }
 
 const user = new Registration();
```

Comment:
group your variable declarations

Reply:
can't group variable declarations if you only have one variable declaration *Roll Safe*